.travis.yml

```diff
@@ -6,6 +6,7 @@ install:
 go:
 - 1.9.x
 - 1.10.x
+- 1.11
```

Comment:
This should be changed to `1.11.x` once the first patch release for Go 1.11 is out (erm, unless I remember incorrectly, but I seem to recall `.x` didn't work if there's no patch release yet)

Reply:
`1.11.x` seems to work well for several of my repos, FYI.

There were some problems with Go 1.10, since YAML and some other code parsed `1.10` a a number (`1.1`), so perhaps that is what you are recalling?

Reply:
Remember that 1.10 is a JSON number, which is identical to 1.1 -- for an actual 1.10 you want "1.10" to force it to a string!